davaleban9.js

```diff
@@ -0,0 +1,106 @@
+
+//1. დაწერეთ ფუნქცია რომელსაც გადასცემ ორ რიცხვს (m,n) და დააბრუნებს რომელიც მეტია
+ //იმ რიცხვს , თუ რიცხვები ტოლია დააბრუნებს 0 -ს
+
+/*function romeliameti(m,n){
+    if (m>n)
+        return m
+    if (m<n)
+        return n
+    else return 0
+}let cvladi=romeliameti(45,60)
+console.log(cvladi)*/
+
+
+//------------------------------------------------------------------------------------
+
+
+//2. დაწერეთ ფუნქცია რომელსაც გადასცემთ ორ ცვლადს(რიცხვს), დააბრუნეთ რიცხვების ჯამი
+
+/*function jami(n,m){
+    return n+m
+}
+console.log (jami(4,13))*/
+
+
+//------------------------------------------------------------------------------------
+
+
+// 3. დაწერეთ ფუნქცია რომელიც დააბეჭდავს თქვენ სახელს და გვარს ( ფუნქციას არ აქვს პარამეტრი)
+
+/*function fullName(){
+    let fullName=("dimitri tsotsonava")
+ console.log(fullName)
+}fullName()*/
+
+
+
+//------------------------------------------------------------------------------------
+
+/*4. დაწერეთ ფუნქცია ორი პარამეტრით : firstname და lastname დააბრუნეთ სრული
+ სახელი და დაბეჭდეთ ( ფუნქცია აბრუნებს სრულ სახელს და ფუნქციის გარეთ ბეჭდავთ)*/
+
+
+/*function sruliSaxeli(firstname,lastname){
+    return firstname+lastname
+
+}let sruli = sruliSaxeli("dimitri ","tsotsonava")
+console.log(sruli)*/
+
+
+
+//------------------------------------------------------------------------------
+
+
+
+/*5. Დაწერეთ ფუნქცია რომელსაც გადასცემთ ერთ რიცხვს (n) და ფუნქციამ უნდა
+დააბრუნოს 1 დან ამ რიცხვამდე  (n-მდე) რიცხვების ნამრავლი ( ანუ თუ რიცხვი არის
+     3 ფუნქციამ უნდა დააბრუნოს 1*2*3 -ის მნიშვნელობა, დაგჭირდებათ ახალი ცვლადის
+      გამოცხადება ფუნქციის შიგნით რომელსაც მიანიჭებთ ნამრავლის მნიშვნელობას. Პ.ს.
+       გამოიყენეთ თქვენთვის სასურველი ციკლი)*/
+
+
+/*function namravli(n){
+    i=1
+    for (i=1;i<n;i++)
+        pasuxi=i*i++
+       return  pasuxi
+ }  console.log(namravli(5))*/
+        
+
+
+//------------------------------------------------------------------------------
+
+//6. დაბეჭდეთ  სტუდენტის სრული სახელი (სახელი+გვარი-  გამოიყენეთ student ობიექტის მეთოდი)
+
+/*let student= {
+    firstname:"dimitri",
+    lastname:"tsotsonava",
+    age:30
+
```

Comment:
fullName : function(){
return this.firstname + " " + this.lastname;
)

console.log(person.fullName())